packages/waterfall/package.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "@mui-internal/waterfall",
```

```suggestion
  "name": "@mui/internal-waterfall",
```
Would this be better? To have the same org name

---

Sure, I'm OK with both.

---

There are other packages that use the @mui-internal scope. I'll rename them in a separate PR.